.buildkite/scripts/steps/check-ci.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common.sh
+
+echo "--- Check CI"
+go version
+mage --version
+BEAT_VERSION=$(make get-version)
+echo "Beat version: $BEAT_VERSION"
```

Comment:
we could prefix this with `---` so this provides a better experience with BK logs.

Reply:
The previous `echo "--- Check CI"` already has this prefix. Isn't it enough?

Reply:
I don't know, what is this `echo`?